grails-app/services/org/pih/warehouse/core/DocumentTemplateService.groovy

```diff
@@ -45,8 +46,12 @@ class DocumentTemplateService {
         Context context = new Context()
         context.putVar("invoiceItems", shipmentInstance?.shipmentItems)
         context.putVar("datePrinted", Constants.EUROPEAN_DATE_FORMATTER.format(new Date()))
-        context.putVar("requisition", shipmentInstance?.requisition?.requestNumber ?: '')
-        JxlsHelper.getInstance().processTemplateAtCell(inputStream, outputStream, context, "Result!A1")
+        context.putVar("requisition", shipmentInstance?.requisition)
+        context.putVar("origin", shipmentInstance?.origin)
+        context.putVar("destination", shipmentInstance?.destination)
+        context.putVar("originRequisitionCount", requisitionService.getRequisitionCountInCurrentFiscalYear(shipmentInstance?.origin))
+        context.putVar("destinationRequisitionCount", requisitionService.getRequisitionCountInCurrentFiscalYear(shipmentInstance?.destination))
+        JxlsHelper.getInstance().processTemplateAtCell(inputStream, outputStream, context, "Template!A1")
```

Comment:
Template!A1 should probably be configurable. Just add something like the following to Config.groovy.
```
openboxes.documentTemplate.invoiceTemplate.targetCell = "Template!A1"
```
The "Template" part scares me. Is there no way to use the default tab's name?

Reply:
Do you mean the `Sheet1`?

Reply:
Yes

Reply:
Ok, will do it